src/account/BaseSmartContractAccount.ts

```diff
@@ -158,14 +158,16 @@ export abstract class BaseSmartContractAccount<
    * @param params -- Typed Data params to sign
    */
   async signTypedDataWith6492(
+    // @ts-ignore
     params: SignTypedDataParams
   ): Promise<`0x${string}`> {
-    const [isDeployed, signature] = await Promise.all([
-      this.isAccountDeployed(),
-      this.signTypedData(params)
-    ])
+    throw new Error("signTypedDataWith6492 not supported")
```

Comment:
why is this commented out?
should we remove this method all together since it is custom name